### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KabohaJeanMark**
**Add media queries to change and adapt layout on different pages.**-

- Apply styling for phone, tablet and laptop screen sizes.
- Use fluid measurements such as percentages for width and vh and vw.
- Use flexbox and change layout for different screen sizes

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

Good job here you've done pretty well using semantic HTML.

However, there are a few issues to be fixed.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

**README**
- [ ] The live link doesn't work kindly fix it.

### **HEADER**

- [ ] On the desktop screen there are nav links on the header please include them and remove the hamburger menu on the desktop screen.
- [ ] Please add the TNW logo at the left top corner of the header as it appears on the original.
- [ ] Also, on all screen sizes ensure that it sticks to the top when the user scrolls.
- [ ] The hamburger menu on the header should only appear on mobile and tablet screens.
- [ ] Also on mobile and tablet screens the TNW logo appears a the top left corner of the header, please include it.

**### Highlights (top 3 articles)**

On the desktop screen size for the Highlights section please take note of the following three points and fix them:
* [ ] The TWN logo appears over the first article, at the top left corner
* [ ] There is a red overlay with a gradient in the top half of the first article
* [ ] The date appears over the first article, at the top right corner

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Comment by tanzila-abedin** [comment]
Hello 
Thank you for the feedback.
We have made the changes you asked for. Kindly have a look.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Tanzila Abedin & Marcus Kaboha** 
Good job with the improvements for this project :muscle:  :muscle:
Your page look similar to the original in the desktop screen size :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [ ] Yellow section (close to the top) most have a bigger bottom and top padding
- [ ] For the mobile version, it should appear the logo in the leftmost part of your navbar
- [ ] For the mobile version, it should appear a burger icon in the rightmost part of your navbar
- [ ] Use the same breakpoints of the original
- [ ] About the three top articles, use "hx" (h1,h2...) tag and "a" tag for the title of these articles
- [ ] use "hx" (h1,h2...) tag and "a" tag for the title of this section
- [ ] For each item in the lastest funding section: use "a" tag for the bottom text, use "a" tag for the two items listed, and use "hx" (h1,h2...) tag and "a" tag for the title of this items.
- [ ] In the footer, all the social media are centered using nav tag
- [ ] In the footer, the links below the social media are centered using a nav tag
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Comment by tanzila-abedin** [comment]
Thank you for your feedback @andresporras3423 .

We have worked on resizing and restructuring the yellow nav-bar according to the project scope. Most of the requested changes have been implemented.
However, the rest of the changes are according to the live site but not according to specifications given by  Microverse for this project. 
Thank you.
project-instructions
project-requirements

**Comment by tanzila-abedin** [comment]
Hello, we have made the necessary changes. kindly review our project.

File: index.html
Comment (teekaytech) [inline index.html]: In the Highlights (top 3 articles) section, 
- The title of each article should use a header (h1...h6) tag and has a link (a tag). Kindly make the changes. i.e.
```html
<h1>
  <a href="#">
    Opal cuts of your apps' internet access to help you stop doomscrolling
  </a>
</h1>
```

File: index.html
Comment (teekaytech) [inline index.html]: The title of this section should use a header (h1...h6) tag. i.e. 
```
<h3>
  <a href="#">
    Latest Funding Rounds On Index.Co 
    <i class="fa fa-angle-right" aria-hidden="true"></i>
  </a>
</h3>
```

File: index.html
Comment (teekaytech) [inline index.html]: The nav menu (See more, Get updates, etc.) are grouped in a `<nav>` tag, not a `<div>` tag.

File: index.html
Comment (teekaytech) [inline index.html]: These items should be linked (i.e. wrapped inside the `<a>` tag) for all the articles in this section.

File: index.html
Comment (teekaytech) [inline index.html]: These social media icons should be centered in a `nav` tag, not a `div` tag.

File: index.html
Comment (teekaytech) [inline index.html]: These menu items should also be centered in a `nav` tag, not a `div` tag.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__Thank you for making most of the required changes highlighted by the previous reviewers :handshake:__

- Your responsiveness is so sweet :+1: 
- You have a clean and readable codebase :100: 

__However, some changes are required before this milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] Do not commit files that are not related to the project (e.g. `.vscode`). Instead, add them to [the `.gitignore` file](https://help.github.com/en/github/using-git/ignoring-files#create-a-local-gitignore).
- [ ] The remaining changes are highlighted in the inline comments below. 
- Most of these changes are semantically related. 
- It is advisable to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) tags so that your project will be meaningful to the browser, and not just the viewers.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: index.html
Comment (KabohaJeanMark) [reply index.html]: Thanks for clearly explaining and making a suggestion. This has been worked on by us.

File: index.html
Comment (KabohaJeanMark) [reply index.html]: Thanks. All link groupings have been grouped in nav instead of div tags

File: index.html
Comment (KabohaJeanMark) [reply index.html]: Done. Thanks for clear feedback.

**Comment by KabohaJeanMark** [comment]
Hello :wave: @teekaytech 

Thanks for the clear feedback for improvements.
We have worked on all of it  using the proper semantic HTML tags for the headings and the nav tags for those link groupings 💪🏾

**Comment by tanzila-abedin** [comment]
Hello wave @teekaytech

Thanks for the clear feedback for improvements.
We have worked on all of it using the proper semantic HTML tags for the headings and the nav tags for those link groupings 💪🏾

**Pull request merged**
